api/src/main/java/cz/muni/fi/pa165/facade/UserFacade.java

```diff
@@ -25,5 +26,17 @@
 
     public void updateUser(UserDTO role);
 
+    public List<UserDTO> getAllUsersByRoleId(Long roleId);
+
+    public boolean authenticateUser(UserDTO user, String plainPassword);
+
+    public Boolean isAdmin(UserDTO user);
+
+    public Long createUser(CreateUserDTO newUser, String plainPassword);
```

Comment:
Heslo ma byt v CreateUserDTO. Pred mapovanim na User entoty ho jen uloz do promenne, pokud je to problem